ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliWeb3ProviderTrustStore.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Given i'm deleting this class, I wouldn't be against you just leaving this one out of your refactoring. the entire class will go with ES-88...